sdk/storage/storage-file/review/storage-file.api.md

```diff
@@ -0,0 +1,1261 @@
+## API Report File for "@azure/storage-file"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
```

Comment:
Is API extractor invoked in build time? Would better to ensure this tool is embedded properly.

Reply:
We can add it as part of the build like keyvault libraries did. The problem was api-extractor never worked on storage code base before this change.

Reply:
logged #5641